README.md

```diff
@@ -20,6 +20,13 @@ was written correctly and much more.
 
 ![Etcher](https://raw.githubusercontent.com/resin-io/etcher/master/screenshot.png)
 
+Supported Operating Systems
+---------------------------
+
+- GNU/Linux
```

Comment:
I realise that "GNU/Linux" is the official nomenclature, but in common parlance just "Linux" is much more common, and is probably more familiar to the people that we're aiming Etcher at?

(the typical Ubuntu user might know that Ubuntu is a version of Linux, but they might think that "GNU/Linux" is some other version of Linux?)